README.md

```diff
@@ -1373,6 +1399,40 @@ console.log(chatResp);
 // }
 ```
 
+`chatCompletion` is based on the [OpenAI Chat Completion](https://platform.openai.com/docs/api-reference/chat) format, and is useful if OpenAI-compatible responses. However, it has limited functionality compared to the standard `chat` method. Read more [here](https://docs.pinecone.io/reference/api/2025-01/assistant/chat_completion_assistant).
+
+```typescript
+import { Pinecone } from '@pinecone-database/pinecone';
+const pc = new Pinecone();
+const assistantName = 'test1';
+const assistant = pc.Assistant(assistantName);
+const chatResp = await assistant.chatCompletion({
+  messages: [
+    {
+      role: 'user',
+      content: 'What is the capital of France?',
+    },
+  ],
+});
+console.log(chatResp);
+// {
+//  id: '000000000000000023e7fb015be9d0ad',
+//  choices: [
+//    {
+//      finishReason: 'stop',
+//      index: 0,
+//      message: {
+//        role: 'assistant',
+//        content: 'The capital of France is Paris.'
+//      }
+//    }
+//  ],
+//  finishReason: 'stop',
+//  model: 'gpt-4o-2024-05-13',
+//  usage: { promptTokens: 493, completionTokens: 38, totalTokens: 531 }
+// }
```

Comment:
Maybe change this to a separate `json` block noting that it's the response shape.